code/datums/supplypacks.dm

```diff
@@ -411,6 +411,25 @@ WEAPONS
 	containername = "\improper M402 mortar crate"
 	group = "Weapons"
 
+/datum/supply_packs/detpack
+	name = "detpack explosives crate (x6)"
+	contains = list(
+					/obj/item/device/radio/detpack,
+					/obj/item/device/radio/detpack,
+					/obj/item/device/radio/detpack,
+					/obj/item/device/radio/detpack,
+					/obj/item/device/radio/detpack,
+					/obj/item/device/radio/detpack,
+					/obj/item/device/assembly/signaler,
+					/obj/item/device/assembly/signaler,
+					/obj/item/device/assembly/signaler,
+					)
+	cost = RO_PRICE_NORMAL
```

Comment:
`RO_PRICE_PRICY`

Reply:
I think you over-estimate the value of these detpacks.

Reply:
Maybe. Testing will tell.